src/Components/WebAssembly/Build/src/targets/Publish.targets

```diff
@@ -24,7 +24,8 @@
 
   <Target
       Name="_BlazorCopyStandaloneWebConfig"
-      AfterTargets="_BlazorCleanupPublishOutput;ComputeResolvedFilesToPublishList">
+      AfterTargets="_BlazorCleanupPublishOutput;ComputeResolvedFilesToPublishList"
+      Condition="!Exists('$(PublishDir)web.config')">
```

Comment:
Doesn't this assume that the web.config already exists in the publish directory before publishing has copied files over?

Reply:
Hmm, it's a good question, but if that's the case wouldn't our tests be failing in CI? They should be publishing from a clean state.